README.md

```diff
@@ -20,6 +22,20 @@ or
 
     ytdl -o "{author.name} - {title}" http://www.youtube.com/watch?v=_HSylqgVYQI
 
+
+Download video and convert to mp3 (Requires ffmpeg)
+
+```bash
+name=`echo myfilename` && ytdl https://www.youtube.com/watch?v=_HSylqgVYQI > $name.mp4 && ffmpeg -i $name.mp4 -b:a 192K -vn $name.mp3
+```
+
+Download video pipe directly intp ffmpeg if you only want the .mp3 file (Requires ffmpeg)
+
+```bash
+name=`echo myfilename` && ytdl http://www.youtube.com/watch?v=_HSylqgVYQI | ffmpeg -i pipe:0 -b:a 192K -vn $name.mp3
```

Comment:
I think most of the time, if people want to convert to mp3, they will want to keep only the mp3 file. I think it's ok to only include this example.

Also, using `name` seems superfluous in this case, because it's only used once.